src/plugins/vis_type_timelion/server/series_functions/fixtures/tl_config.js

```diff
@@ -30,11 +30,7 @@ export default function () {
       if (!functions[name]) throw new Error('No such function: ' + name);
       return functions[name];
     },
-    esDataClient: sinon.stub().returns({
-      callAsCurrentUser: function () {
-        return Promise.resolve(esResponse);
-      },
-    }),
+    esDataClient: sinon.stub().returns(Promise.resolve({ rawResponse: esResponse })),
```

Comment:
It seems there is no `esDataClient` in `tlConfig` anymore.

Reply:
done